src/connector/builder.rs

```diff
@@ -174,7 +174,7 @@ impl ConnectorBuilder<WantsProtocols1> {
         })
     }
 
-    /// Enable all HTTP versions built into this library
+    /// Enable all HTTP versions built into this library (enabled with Cargo features)
```

Comment:
small nit: I think this change would ideally be squished into the first commit since its unrelated to the test coverage.

Reply:
We'll just squash merge this nit away.